webapp/src/commoncomponents/imageGrid.tsx

```diff
@@ -34,87 +32,54 @@ const useStyles = makeStyles((theme: Theme) =>
 
 export default function ImageGrid(props: any) {
```

Comment:
Toto in future: type props (there is an easy way to do it just with parameter names and types and hard way with validation, required and optional etc). You can add "Refactor" "Low priority" issue about it.